Rakefile

```diff
@@ -1180,7 +1180,7 @@ end
 
 def decompile_apk (file, destination)
   puts "Decompiling APK #{file} to #{destination}..."
-  sh "java -jar tools/apktool/apktool.jar d -f #{file} #{destination}"
+  sh "java -jar tools/apktool/apktool.jar d -f #{file} -o #{destination}"
```

Comment:
The newer `apktool.jar` is bundled along with this change, so if you get this change, you also get the newer apktool. But this is really a merged in change that we haven't landed yet (#316) which we probably should.